recommendations_Infra_demo/recommendations_demo/kruize/json_validate.py

```diff
@@ -0,0 +1,165 @@
+"""
+Copyright (c) 2022, 2022 Red Hat, IBM Corporation and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import jsonschema
+from jsonschema import validate, draft7_format_checker
+
+DIRECTIONS_SUPPORTED = ("maximize", "minimize")
+
+DIRECTION_NOT_SUPPORTED = "Direction not supported!"
+JSON_NULL_VALUES = ("is not of type 'string'", "is not of type 'integer'", "is not of type 'number'")
+VALUE_MISSING = " cannot be empty or null!"
+
+exp_input_schema = {
```

Comment:
I'd suggest maintaining this as a JSON file and we can read from it here. This way it is easier to keep track changes to the JSON

Reply:
Will take it up as a different PR as the same needs to changed in remote_monitoring tests  as well.